projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -470,8 +471,9 @@
 		}
 	}
 
-	.jetpack-field-option {
-		&.field-option-radio .jetpack-option__type {
+	.jetpack-field-option.field-option-radio,
+	.wp-block-jetpack-field-option-radio {
+		.jetpack-option__type {
```

Comment:
Selector needs to be updated because of https://github.com/Automattic/jetpack/pull/39141